### DISCUSSION TITLE
pakage

### DISCUSSION THREAD

**Discussion by Swanboss01** [community/community#145438] [Packages]
### Select Topic Area
how to create package APK and EXE

### Body

Hello world today my question is : how to create package APK and EXE in github

**Comment by ebndev** [comment]
Hi @Swanboss01, thanks for being a part of the GitHub Community!

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" as GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. Any repeat and future violations may result in a temporary or indefinite block from the Community. 

We appreciate your understanding.
